src/main/resources/logback.xml

```diff
@@ -57,7 +57,7 @@
         <appender-ref ref="json"/>
     </logger>
 
-    <logger name="org.constellation" level="debug">
+    <logger name="org.constellation" level="info">
```

Comment:
Please leave it as debug

Reply:
we do know about the issue with ELK so till it will not be resolved let's leave it, you can always change it before triggering cluster.